Answers/hw03_stubs_do_not_edit.py

```diff
@@ -0,0 +1,85 @@
+
+# TODO Do not edit this file directly!
+# TODO Instead, create a new file called hw03_username.py and copy this code into it!
+
+#################################################################################
+# Author: Megan Lucas
+# Username:Megan_Lucas
+#
+# Assignment:HW03_stub
+# Purpose: To build a bouquet of flowers with a bow
+# Google Doc Link:https://docs.google.com/document/d/1ZtouqxWRp3KCr7y-horbUReL1Hd0j0SWmor7oxk6oxs/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+
+
+import turtle
+
+wn = turtle.Screen()
+wn.bgcolor("lightblue")
+m = turtle.Turtle()
+
+import turtle
```

Comment:
This is a repeat of the `import turtle` above and can be removed.

Reply:
Usually this means code was copy and pasted from somewhere, yet I see nothing in the Acknowledgements section...